### PR TITLE
Update severity.md

### PR DESCRIPTION
**Pull request by coapacetic**
## Description & motivation
Hi team! I was looking at this page during a training today and noticed that some of the language may be confusing with the latest change to generic and singular tests.  I used the word 'built-in' which might need some wordsmithing.  Happy to chat more about updating this page.

## To-do before merge
[ ] Review the updated language for generic and singular tests to align with the latest changes
[ ] Modify or keep the use of the world 'built-in' for the 4 schema tests that ship with dbt.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [X] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/reference/resource-configs/severity.md [hunk 1]
```diff
@@ -29,18 +29,18 @@ Here's how those play in practice:
 Note that test warn statuses will return errors instead if the [`--warn-error`](global-cli-flags#warnings-as-errors) flag is passed. Unless dbt is told to treat warnings as errors, a test with `warn` severity will never return an error.
 
 <Tabs
-  defaultValue="specific"
+  defaultValue="generic"
   values={[
-    { label: 'Specific test', value: 'specific', },
-    { label: 'One-off test', value: 'one_off', },
-    { label: 'Generic test block', value: 'generic', },
+    { label: 'Built in generic tests', value: 'generic', },
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/severity.md]: We use "[out of the box" if you scroll down here](https://deploy-preview-1011--docs-getdbt-com.netlify.app/docs/building-a-dbt-project/tests#generic-tests), so I wonder if we should stick with that and say "out-of-the-box generic tests," though it is longer 🤔

**Review by runleonarun — APPROVED** [review]
@coapacetic Thank you for catching this! Just one non-blocking comment. I suggested using "out-of-the-box generic tests" instead of "built-in generic tests."

File: website/docs/reference/resource-configs/severity.md (hunk 1 above)
Comment (coapacetic) [reply website/docs/reference/resource-configs/severity.md]: sounds good to me!

**Comment by coapacetic** [comment]
Thanks for merging Leona! Sorry for leaving that one lingering.

**Pull request merged**